tasks/interactive-bots.md

```diff
@@ -263,16 +261,16 @@ requirements.
 
 ### Task Type D: Create your own bot
 
-We recommend claiming this task after completing Task Type A and Task Type B.
+We recommend claiming this task type after completing Task Type A and Task Type B.
 
 * Come up with an idea for your own interactive bot that would improve Zulip.
-Make sure to make clear what would be expected of the bot -  what messages
-should it react to, what should it do with the message content and where should
-it post any output.
+Make sure to make clear what would be expected of the bot - the kind of messages
+it should react to, what it should do with the message content and where it
+should post any output.
 
 * Present your idea for an interactive bot on the `interactive bots` topic on
 the `GCI tasks` stream. Consult with mentors and make sure your idea is approved
- before you start coding.
+ before you start working on it.
```

Comment:
Communication is also working on the task. Coding is implementing a bot you've agreed on with a mentor. I think we can change to "implementing" but not "working".